### DISCUSSION TITLE
How old is the Ring camera snapshot provided in the integration?

### DISCUSSION THREAD

**Discussion by dbergqui** [tsightler/ring-mqtt#396] [General Discussion]
First, the compliment - Great integration. When I pull the standard image and compare with yours, it matches the snapshot I see in the ring app whereas the image shown in the standard Ring integration seems to be from the last event.

Then the Question: how old is the snapshot? When I view the Ring app on my phone it shows an overlay counter with how old the snapshot is (xx sec. ago). Is that information available through your integration or somewhere else so I can tell that I am not getting updates? (I haven't had a failure yet - maybe if the MQQT stream gets broken or the integration fails to get a new image I will get a BSOD image and I could assume that no updates are occurring?).

If the time of the snapshot were available I would just include that in my display card. Right now I use the last event from Ring and the Snapshot from you.

Thanks!

**Comment by tsightler** [comment]
The snapshot camera includes a timestamp attribute in standard epoch format with 1 second granularity so that should be what you are looking for.
